tests/dataimportcron_test.go

```diff
@@ -367,6 +368,120 @@ var _ = Describe("DataImportCron", func() {
+		dv, err = utils.CreateDataVolumeFromDefinition(f.CdiClient, ns, dv)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Wait for import completion")
+		err = utils.WaitForDataVolumePhase(f, ns, cdiv1.Succeeded, dv.Name)
+		Expect(err).ToNot(HaveOccurred(), "Datavolume not in phase succeeded in time")
+
+		By(fmt.Sprintf("Verify pvc was created %s", dv.Name))
+		pvc, err := utils.WaitForPVC(f.K8sClient, ns, dv.Name)
+		Expect(err).ToNot(HaveOccurred())
+		By(fmt.Sprintf("Verify DataImportCronLabel is passed to the pvc: %s", pvc.Labels[common.DataImportCronLabel]))
+		Expect(pvc.Labels[common.DataImportCronLabel]).To(Equal(cronName))
+
+		pvc.Labels[common.DataImportCronLabel] = ""
+		By("Update DataImportCron label to be empty in the pvc")
+		_, err = f.K8sClient.CoreV1().PersistentVolumeClaims(pvc.Namespace).Update(context.TODO(), pvc, metav1.UpdateOptions{})
+		Expect(err).ToNot(HaveOccurred())
+
+		By(fmt.Sprintf("Create new DataImportCron %s, url %s", cronName, *reg.URL))
+		cron = utils.NewDataImportCron(cronName, "5Gi", scheduleEveryMinute, dataSourceName, importsToKeep, *reg)
+		retentionPolicy := cdiv1.DataImportCronRetainNone
+		cron.Spec.RetentionPolicy = &retentionPolicy
+		cron, err = f.CdiClient.CdiV1beta1().DataImportCrons(ns).Create(context.TODO(), cron, metav1.CreateOptions{})
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Wait for conditions")
+		waitForConditions(corev1.ConditionFalse, corev1.ConditionTrue)
+
+		By("Verify CurrentImports update")
+		currentImportDvName := cron.Status.CurrentImports[0].DataVolumeName
+		Expect(currentImportDvName).ToNot(BeEmpty())
+
+		dataSource, err := f.CdiClient.CdiV1beta1().DataSources(ns).Get(context.TODO(), dataSourceName, metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+
+		if pvc := dataSource.Spec.Source.PVC; pvc != nil {
+			By(fmt.Sprintf("Verify pvc was created %s", pvc.Name))
+			_, err = utils.WaitForPVC(f.K8sClient, ns, pvc.Name)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Wait for import completion")
+			err = utils.WaitForDataVolumePhase(f, ns, cdiv1.Succeeded, pvc.Name)
+			Expect(err).ToNot(HaveOccurred(), "Datavolume not in phase succeeded in time")
+		} else if snap := dataSource.Spec.Source.Snapshot; snap != nil {
+			By("Wait for snapshot to be ready")
+			snapshot := &snapshotv1.VolumeSnapshot{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      snap.Name,
+					Namespace: snap.Namespace,
+				},
+			}
+			snapshot = utils.WaitSnapshotReady(f.CrClient, snapshot)
+			By("Verify snapshot is ready to be used")
+			Expect(snapshot.Status.ReadyToUse).ToNot(BeNil())
+			Expect(*snapshot.Status.ReadyToUse).To(BeTrue())
+		}
+
+		By("Check garbage collection")
+		Eventually(func() int {
+			dvList, err := f.CdiClient.CdiV1beta1().DataVolumes(ns).List(context.TODO(), metav1.ListOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			return len(dvList.Items)
+		}, dataImportCronTimeout, pollingInterval).Should(Equal(0), "Garbage collection failed cleaning old imports")
```

Comment:
I think this is why the NFS lane fails;
(With non-snapshot-capable storage) You will always have 1 main (non-GCd) datavolume present (latest pull) 
so you will never reach `len(dvList.Items) == 0`

Maybe using [test_id:7406] will make life easier and test runtime lower, but I am not sure,
let me know what you think

Reply:
I will take a look at `[test_id:7406] ` and see if I can integrate the test there instead.
@arnongilboa wdyt?